testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -250,3 +249,34 @@
     _out, _code = $server.run('cobbler sync')
   end
 end
+
+When(/^I start local monitoring of Cobbler$/) do
+  cobbler_conf_file = '/etc/cobbler/logging_config.conf'
+  cobbler_log_file = '/var/log/cobbler/cobbler_debug.log'
+  $server.run("rm #{cobbler_log_file}", check_errors: false)
+  _result, code = $server.run("test -f #{cobbler_conf_file}.old", check_errors: false)
+  if !code.zero?
+    handler_name = 'FileLogger02'
+    handler_class = "\"\n[handler_#{handler_name}]\n" \
+                  "class=FileHandler\n" \
+                  "level=DEBUG\n" \
+                  "formatter=Logfile\n" \
+                  "args=('#{cobbler_log_file}', 'a')\""
+    command = "cp #{cobbler_conf_file} #{cobbler_conf_file}.old && " \
+              "line_number=`awk \"/\\\[handlers\\\]/{ print NR; exit }\" #{cobbler_conf_file}` && " \
+              "sed -e \"$(($line_number + 1))s/$/,#{handler_name}/\" -i #{cobbler_conf_file} && " \
+              "line_number=`awk \"/\\\[logger_root\\\]/{ print NR; exit }\" #{cobbler_conf_file}` && " \
+              "sed -e \"$(($line_number + 2))s/$/,#{handler_name}/\" -i #{cobbler_conf_file} && " \
+              "echo -e #{handler_class} >> #{cobbler_conf_file}"
+    $server.run("#{command} && systemctl restart cobblerd", check_errors: false)
```

Comment:
I wonder if we should check for errors here in case a restart of Cobbler fails? What do you think? Same 2 lines below in the `else` section.

Reply:
It would make sense to check for errors in case of a Cobbler failure, it would help catch issues faster if there is a configuration issue.

Reply:
You can btw use `cobbler-settings` on the server to validate the configuration of Cobbler without starting the daemon.

Reply:
@elariekerboull is my suggestion in the context of this PR or do you want to get this merged? @nodeg what other kinds of errors do you have in mind? There are plenty of errors that can occur. Another instance of Cobbler already binding to the ports required only being on of them.

Reply:
I am looking into incorporating this validation, however I believe I might add it in another PR I am currently working on and merge this one for now.